### PR TITLE
Dynamic adjustment RSSI Threshold

### PR DESCRIPTION
**Pull request by JohnOH**
Initially is very sensitive 0xFF.

### CODE REVIEW

File: src/driver/rf69.h [hunk 1]
```diff
@@ -198,11 +216,35 @@ int RF69<SPI>::receive (void* ptr, int len) {
                 afc = readReg(REG_AFCMSB) << 8;
                 afc |= readReg(REG_AFCLSB);
 #endif
+
+                printf(" (%d%s%d:%d T%d:%d:%d)%d\n",
```
Comment (jcw) [inline src/driver/rf69.h]: no printf's inside the driver, please

**Comment by jcw** [comment]
is this for testing or should this be part of the official driver (once we get there)?

**Comment by JohnOH** [comment]
Martyn and I think something with this capability will be required
    in the official driver - less all the debug printf's of course.
    
    John.
    On 10/02/2015 19:44, Jean-Claude
      Wippler wrote:
    
    
      is this for testing or should this be part of the official
        driver (once we get there)?
      —
        Reply to this email directly or view
          it on GitHub.

**Comment by JohnOH** [comment]
Also, it would be helpful if the IRQ1_RXREADY at least was put into
    interrupt control as I have a number of timing issues where I missed
    the RSSI etc but went straight into IRQ2_PAYLOADREADY. I put this
    down the scanning time. I would be starting on page one for
    interrupts for the LPC.
    
    John.
    --
    On 10/02/2015 19:44, Jean-Claude
      Wippler wrote:
    
    
      is this for testing or should this be part of the official
        driver (once we get there)?
      —
        Reply to this email directly or view
          it on GitHub.

**Pull request closed without merging**